src/main/java/com/notably/commons/core/path/RelativePath.java

```diff
@@ -1,53 +1,112 @@
 package com.notably.commons.core.path;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
 import com.notably.commons.core.path.exceptions.InvalidPathException;
 
 /**
  * TODO: Add Javadoc
  */
 public class RelativePath implements Path {
+    public static final String INVALID_RELATIVE_PATH = "Invalid relative path";
+    public static final String VALIDATION_REGEX = "((\\.|\\..|\\p{Alnum}+))+\\/?";
+    private final List<String> paths;
+    private RelativePath(String relativePathString) {
+        this.paths = new ArrayList<>();
+        for (String obj : relativePathString.split("/")) {
+            if (!obj.trim().equals("")) {
+                this.paths.add(obj.trim());
+            }
+        }
+    }
```

Comment:
@ljiazh3ng wanna discuss with you about this!

Say if our user inputs the `"CS2103/../CS2103/hello"` string to this constructor. Would you prefer if we:
1. Keep it as is. That is, the components will content ["CS2103", "..", "CS2103", "hello"], or
2. Simplify the path. That is, we do something similar to the thing we do in`AbsolutePath.fromRelativePath` and store the components as ["hello"] (since `"CS2103/../CS2103"` cancels each other out)

Which one?